plugins/inputs/aliyuncms/aliyuncms_test.go

```diff
@@ -69,6 +69,36 @@ func (m *mockGatherAliyunCMSClient) DescribeMetricList(request *cms.DescribeMetr
 	return resp, nil
 }
 
+type mockGatherAliyunRDSClient struct{}
+
+func (m *mockGatherAliyunRDSClient) DescribeDBInstancePerformance(request *rds.DescribeDBInstancePerformanceRequest) (
+	*rds.DescribeDBInstancePerformanceResponse, error) {
+	resp := new(rds.DescribeDBInstancePerformanceResponse)
+
+	switch request.Key {
+	//TODO Adapt the Tests and the Mock
```

Comment:
What's left here?

Reply:
It is my quality aspect to write new unit tests for the delivered new function. I am currently working on the mocks and will further customize the tests today.